### DISCUSSION TITLE
Bring player

### DISCUSSION THREAD

**Discussion by DeadFinder** [tupoy-ya/TupoyeMenu#87] [Q&A]
Is bring working for anyone? Because for me it absolutely not working, RAGE_JOAAT("VEHICLE_RCBANDITO") not founding RCBANDITO and i'm replaced it with RCBANDITO hash, but how i'm not tried the player what i'm wanted to bring with using teleport_player_to_coords doesn't sit to vehicle, maybe some suggestions? Currently as i'm found (!hnd) and (!g_pointers->m_gta.m_handle_to_ptr(hnd)->m_net_object) not passes and blocks execution of code, also i dont understand how to force player to sit in to car. 

So another question for @tupoy-ya, maybe there is any way to contact you through discord?

**Comment by SlavicBlin** [comment]
have you tried it in Yimmenu latest build?

**Reply by DeadFinder** [reply]
Nope, but i'm checked code and probably its same of vehicle spawn and teleport, only one difference as i see
In Yim: auto hnd = vehicle::spawn(VEHICLE_RCBANDITO, 
and in Tupoye: Vehicle hnd = vehicle::spawn(RAGE_JOAAT("VEHICLE_RCBANDITO"),

**Reply by SlavicBlin** [reply]
okay i will give it a try on tupoye

**Reply by SlavicBlin** [reply]
Can confrim it does not work.
I will test it with Yim

**Reply by SlavicBlin** [reply]
works in latest yim build

**Reply by SlavicBlin** [reply]
i will try merge on my own build to see

**Reply by DeadFinder** [reply]
Ok, i will try to replace functions by myself in my local build, thanks for help

**Reply by DeadFinder** [reply]
Also found interesting item, as we can see in vehicle.cpp we have function **void set_mp_bitset(Vehicle veh, bool is_stolen = false)**
but in vehicle.hpp we have only **void set_mp_bitset(Vehicle veh)** ;)

**Reply by SlavicBlin** [reply]
aight yeah no idea honestly i tried merging and manually building with latest yim build it just does not let me build when changing the classes. i hope you can figure it out 👍

**Reply by tupoy-ya** [reply]
YimMenu didn't change anything related to `Bring Player` recently so it's probably just a bad merge by me.

<hr>

> So another question for @tupoy-ya, maybe there is any way to contact you through discord?

There is, but please use matrix instead.
@tupoy_ya:matrix.org

**Reply by DeadFinder** [reply]
So i successfully replaced functions and it works goodly. Thanks @SlavicBlin for help and @tupoy-ya for answering and making this great fork of Yim.

**Comment by tupoy-ya** [comment]
Should be fixed in 257f7d6708c2fe320b3b984128f497bc97b86bfb.
Didn't test.
